api/src/test/java/com/epam/pipeline/controller/billing/BillingControllerTest.java

```diff
@@ -80,38 +80,26 @@ public void setUp() {
 
     @Test
     public void shouldFailGetBillingChartInfoForUnauthorizedUser() throws Exception {
-        mvc().perform(post(GET_BILLING_CHART_INFO_URL)
-                .servletPath(SERVLET_PATH))
-                .andExpect(status().isUnauthorized());
+        controllerTestUtils.postRequestUnauthorized(mvc(), GET_BILLING_CHART_INFO_URL);
     }
 
     @Test
     @WithMockUser
     public void shouldReturnBillingChartInfo() throws Exception {
         Mockito.doReturn(billingChartInfos).when(mockBillingApiService).getBillingChartInfo(billingChartRequest);
 
-        final MvcResult mvcResult = mvc().perform(post(GET_BILLING_CHART_INFO_URL)
-                .servletPath(SERVLET_PATH)
-                .contentType(EXPECTED_CONTENT_TYPE)
-                .content(REQUEST_JSON))
-                .andExpect(status().isOk())
-                .andExpect(content().contentType(EXPECTED_CONTENT_TYPE))
-                .andReturn();
+        final MvcResult mvcResult = controllerTestUtils
```

Comment:
Could you please pass request to method arguments (to my mind it would be more flexible):
```
public MvcResult performRequest(final MockHttpServletRequestBuilder requestBuilder) throws Exception {
        return mvc.perform(requestBuilder
                .servletPath(SERVLET_PATH)
                .contentType(CONTENT_TYPE))
                .andExpect(status().isOk())
                .andExpect(MockMvcResultMatchers.content().contentType(CONTENT_TYPE))
                .andReturn();
    }
```
This way the method calling will be:
```
final MvcResult mvcResult = controllerTestUtils
                .performRequest(post(GET_BILLING_CHART_INFO_URL)
                        .content(REQUEST_JSON));
```